<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterferenceAnalysisDialogBase</class>
 <widget class="QDialog" name="InterferenceAnalysisDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>576</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>576</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Interference Analysis</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon_interference.svg</normaloff>icon_interference.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="leftLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="layerLabel">
           <property name="text">
            <string>Layer:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="layerComboBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="frequencyLabel">
           <property name="text">
            <string>Frequency/EARFCN field:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="frequencyFieldComboBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="pciLabel">
           <property name="text">
            <string>PCI/PSC field:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="pciFieldComboBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="bandLabel">
           <property name="text">
            <string>Band field:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="bandFieldComboBox"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="azimuthLabel">
           <property name="text">
            <string>Azimuth field:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="azimuthFieldComboBox"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="beamwidthLabel">
           <property name="text">
            <string>Beamwidth field:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="beamwidthFieldComboBox"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="siteIdLabel">
           <property name="text">
            <string>Site ID Field:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="siteIdFieldComboBox"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="sectorLabel">
           <property name="text">
            <string>Sector Field:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QComboBox" name="sectorFieldComboBox"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="analysisTypeLabel">
           <property name="text">
            <string>Analysis Type:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <layout class="QVBoxLayout" name="analysisTypeLayout">
           <item>
            <widget class="QCheckBox" name="coChannelCheckBox">
             <property name="text">
              <string>Co-Channel Interference</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="adjacentChannelCheckBox">
             <property name="text">
              <string>Adjacent Channel Interference</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="pciConflictCheckBox">
             <property name="text">
              <string>PCI Conflicts (mod 3/6)</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="interferenceDistanceLabel">
           <property name="text">
            <string>Interference Distance (km):</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QDoubleSpinBox" name="interferenceDistanceSpinBox">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>0.500000000000000</double>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="value">
            <double>2.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="overlapThresholdLabel">
           <property name="text">
            <string>Overlap Threshold (degrees):</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QSpinBox" name="overlapThresholdSpinBox">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>90</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="outputLabel">
           <property name="text">
            <string>Output Layer Prefix:</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLineEdit" name="outputPrefixLineEdit">
           <property name="text">
            <string>Interference</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="mitigationGroupBox">
         <property name="title">
          <string>Mitigation Suggestions</string>
         </property>
         <layout class="QVBoxLayout" name="mitigationLayout">
          <item>
           <widget class="QCheckBox" name="suggestFrequencyCheckBox">
            <property name="text">
             <string>Suggest Frequency Changes</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="suggestTiltCheckBox">
            <property name="text">
             <string>Suggest Tilt Adjustments</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="suggestPciCheckBox">
            <property name="text">
             <string>Suggest PCI Changes</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="infoTextBrowser">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;h3 style=&quot; margin-top:8px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9.5pt; font-weight:600; color:#f44336;&quot;&gt;Interference Analysis&lt;/span&gt;&lt;/h3&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.5pt; font-weight:600;&quot;&gt;Purpose:&lt;/span&gt;&lt;span style=&quot; font-size:8.5pt;&quot;&gt; Identifies and visualizes interference issues between sectors.&lt;/span&gt; &lt;/p&gt;
&lt;h3 style=&quot; margin-top:8px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9.5pt; font-weight:600; color:#f44336;&quot;&gt;Analysis Types:&lt;/span&gt;&lt;/h3&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.5pt; font-weight:600;&quot;&gt;Co-Channel:&lt;/span&gt;&lt;span style=&quot; font-size:8.5pt;&quot;&gt; Same frequency&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.5pt; font-weight:600;&quot;&gt;Adjacent Channel:&lt;/span&gt;&lt;span style=&quot; font-size:8.5pt;&quot;&gt; 5-20 MHz apart&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.5pt; font-weight:600;&quot;&gt;PCI Conflicts:&lt;/span&gt;&lt;span style=&quot; font-size:8.5pt;&quot;&gt; mod3/mod6 issues&lt;/span&gt; &lt;/p&gt;
&lt;h3 style=&quot; margin-top:8px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9.5pt; font-weight:600; color:#f44336;&quot;&gt;Instructions:&lt;/span&gt;&lt;/h3&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.5pt;&quot;&gt;1. Select sector layer&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.5pt;&quot;&gt;2. Map required fields&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.5pt;&quot;&gt;3. Check analysis types&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.5pt;&quot;&gt;4. Set interference distance&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.5pt;&quot;&gt;5. Set overlap threshold (degrees)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.5pt;&quot;&gt;6. Choose mitigation options&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.5pt;&quot;&gt;7. Click Run Analysis&lt;/span&gt; &lt;/p&gt;
&lt;h3 style=&quot; margin-top:8px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9.5pt; font-weight:600; color:#f44336;&quot;&gt;Output:&lt;/span&gt;&lt;/h3&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.5pt;&quot;&gt;Line layer showing interference pairs with severity classification.&lt;/span&gt; &lt;/p&gt;
&lt;h3 style=&quot; margin-top:8px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9.5pt; font-weight:600; color:#f44336;&quot;&gt;Mitigation:&lt;/span&gt;&lt;/h3&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.5pt;&quot;&gt;• Frequency changes&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.5pt;&quot;&gt;• Tilt adjustments (+2 to +5°)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.5pt;&quot;&gt;• PCI reassignment&lt;/span&gt; &lt;/p&gt;
&lt;h3 style=&quot; margin-top:8px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9.5pt; font-weight:600; color:#f44336;&quot;&gt;Tips:&lt;/span&gt;&lt;/h3&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.5pt;&quot;&gt;• Urban: 1-2 km distance&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.5pt;&quot;&gt;• Threshold: 30° typical&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.5pt;&quot;&gt;• Focus on High severity first&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="progressLayout">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="visible">
        <bool>true</bool>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="runButton">
       <property name="text">
        <string>Run Analysis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InterferenceAnalysisDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
